nixos/modules/services/networking/nomad.nix

```diff
@@ -52,12 +52,24 @@ in
       settings = mkOption {
         type = format.type;
         default = {
-          # Agrees with `StateDirectory = "nomad"` set below.
           data_dir = "/var/lib/nomad";
         };
         description = ''
           Configuration for Nomad. See the <link xlink:href="https://www.nomadproject.io/docs/configuration">documentation</link>
           for supported values.
+
+          Notes about `data_dir`:
+
+          If `data_dir` is set to a value other than the default value of
+          `"/var/lib/nomad"` it is the Nomad cluster manager's responsibility
+          to make sure that this directory exist and has the appropriate
+          permissions. One way to do this is with the `ExecStartPre` feature of
```

Comment:
You probably shouldn't suggest that because the code in `ExecStartPre` lacks privileges to `mkdir` anything the user might want. `systemd.tmpfiles.rules` might be a better suggestion. But then again... if the user is deviating from the default it is likely because they have some sort of special setup, like a different partition, etc... and would make the directory themselves.